src/Uno.UI.Runtime.Skia.Wpf/Input/WpfCorePointerInputSource.cs

```diff
@@ -111,7 +111,9 @@ public void ReleasePointerCapture()
 		=> _hostControl.ReleaseMouseCapture();
 
 	#region Native events
-	private void HostOnMouseEvent(InputEventArgs args, TypedEventHandler<object, PointerEventArgs>? @event, [CallerArgumentExpression(nameof(@event))] string eventName = "")
+
```

Comment:
@jeromelaban, @dr1rrb should we add more tests related to these changes?

Reply:
A manual test would be useful, if we don't have any.

Reply:
@jeromelaban  Any update here? Or what should I do? Thank you.

Reply:
@lindexi yes, in general, if testing is difficult because if requires integration with the outer platform, we add a "manual test" page which explains what to do, and what the outcome is. Something like this:

https://github.com/unoplatform/uno/blob/937409d95c680ce453323be2a228c41db79aead5/src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Input/XamlUICommandTests.xaml.cs#L19

Reply:
@jeromelaban Thank you. I've attempted to add a test file, but I'm unsure if I've done it correctly. If I have, how should I write the code for the test file to make the test engineers happier? Would it suffice to just output the parameters to the console?

Reply:
Thanks @lindexi, yes, this is the right way to do it. You can add a listview or a textbox that will concatenate the events so that QA can validate that the events are raised properly.

Reply:
@jeromelaban Thank you very much.

Reply:
@jeromelaban I updated my test files. Could you review my code? Thank you.

Reply:
That will do! Thanks!